README.md

```diff
@@ -1,112 +1,204 @@
+New even-numbered release lines (e.g. v6, v8, v10, etc) are cut in April.
+New odd-numbered release lines (e.g. v5, v7, v9) are cut in October.
+These 6-monthly release lines are initially referred to as "**Current**" and receive most non-breaking changes that occur on the `master` development branch of Node.js, hence their semver-minor version numbers can increase rapidly during this period.
+
+## Current ⇒ LTS
+
+In October of each year, the even-numbered "Current" release line is transitioned on to the Long Term Support (LTS) plan and begin to be referred to as "**LTS**".
+At the same time, a new odd-numbered "Current" release line is created.
+
+## Support timeframes
+
+Every release line covered by LTS will be actively maintained for an initial period of 18 months from the date it enters LTS coverage; this is "**Active LTS**".
+Following those 18 months of active support, the release line will transition into "**Maintenance**" mode for 12 additional months.
+
+Given this schedule, there will be no more than two "Active LTS" release lines at any given time, overlapping for a maximum period of six months.
+Users are encouraged to use this overlap to plan a migration path between LTS release lines.
+
+Release lines not covered by LTS (i.e. odd-numbered release lines) receive an additional 2 months of "Maintenance"-style support following their 6 months as "Current" releases.
+
+## Acceptable changes for release lines
+
+All release lines covering a single semver-major version number, be it labelled "Current", "Active LTS" or "Maintenance" will:
+
+* _Not_ receive commits labelled "[***semver-major***](https://github.com/nodejs/node/pulls?utf8=%E2%9C%93&q=is%3Apr%20label%3Asemver-major%20)" except in the special case of critical security fixes (see below).
+  That is, the external API of any release line shall not be intentionally changed in a non-backward-compatible way.
+* Receive commits labelled "[***semver-minor***](https://github.com/nodejs/node/pulls?utf8=%E2%9C%93&q=is%3Apr%20label%3Asemver-minor%20)".
+  However, it is expected that release lines that are covered by LTS shall receive a minimal number of these changes in order to maximise stability.
+  Additionally, semver-minor changes will be delayed and grouped as much as is practical in LTS release lines in order to ease the upgrade path for users of that line.
+  Any semver-minor commits proposed for inclusion in an LTS release line will require:
+  - sign-off from at least one active member of the LTS Working Group;
+  - no objection from any active member of the LTS Working Group
+* Receive semver-patch commits, i.e. those not labelled either semver-major or semver-minor as appropriate.
+  Any semver-patch commits proposed for inclusion in an LTS release line will require:
+  - sign-off from at least one active member of the LTS Working Group;
+  - no objection from any active member of the LTS Working Group
```

Comment:
At this point I'm ok with landing as is and considering adding a CITGM reference later on.